README.md

```diff
@@ -10,33 +10,115 @@ This repository contains a Terraform provider for the [Materialize platform](htt
 * [Terraform](https://www.terraform.io/downloads.html) >= 1.0.3
 * [Go](https://golang.org/doc/install) >= 1.16
 
-## Developing the provider
+## Installation
 
-If you wish to work on the provider, you'll first need
-[Go](http://www.golang.org) installed on your machine (see
-[Requirements](#requirements) above).
+To use the provider, add the following configuration to your Terraform settings:
 
-To compile the provider, run `make install`. This will build the provider and put the provider binary in the correct location within `~/.terraform.d` so that Terraform can find the plugin.
+```hcl
+terraform {
+  required_providers {
+    materialize = {
+      source = "materialize.com/devex/materialize"
+    }
+  }
+}
+```
+
+Configure the provider by adding the following block to your Terraform project:
+
+```hcl
+provider "materialize" {
+  host     = "materialized_hostname"
+  username = "materialize_username"
+  password = "materialize_password"
+  port     = 6875
+  database = "materialize"
+}
+```
+
+Once you have configured the provider, you can start defining resources using Terraform. You can find examples of how to define resources in the [`examples`](./examples/) directory.
+
+## Usage
+
+### Managing resources
+
+You can manage resources using the `terraform apply` command. For example, to create a new connection named `kafka_connection`, add the following resource definition to your Terraform project:
+
+```hcl
+# main.tf
+resource "materialize_connection_kafka" "kafka_connection" {
+  name = "kafka_connection"
+  kafka_broker {
+    broker = "b-1.hostname-1:9096"
+  }
+  sasl_username = "example"
+  sasl_password {
+    name          = "kafka_password"
+    database_name = "materialize"
+    schema_name   = "public"
+  }
+  sasl_mechanisms = "SCRAM-SHA-256"
+  progress_topic  = "example"
+}
+```
+
+Then, run apply the changes:
```

Comment:
I think you can remove this section since you mention using the `terraform apply` in the paragraph above.